ci_framework/roles/reproducer/README.md

```diff
@@ -23,3 +23,37 @@ This role isn't intended to be called outside of the `reproducer.yml` playbook.
 
 ## Examples
 Please follow the [documentation about the overall "reproducer" feature](https://ci-framework.readthedocs.io/en/latest/cookbooks/reproducer.html).
+
+### Push repositories
```

Comment:
This part would be nice to appear also in general `Reproducer` documentation.
Probably somewhere over here: https://github.com/openstack-k8s-operators/ci-framework/blob/f635b814b0bc9f29345c05ffff7b595ea42b6ff6/docs/source/reproducers/03-zuul.md#how-do-i-run-that-job

Reply:
Sounds good. AFAIR you tried to implement auto-generation of roles description in the documentation. If we could at least point out cross-dependencies that would significantly help and simplify time required to find the important information.

Reply:
done